### ISSUE TITLE
Documentation Skip to Content Missing

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#2239]
It looks like the skip to content component used previously to help keyboard users has been removed in our documentation. Can we re-add it to help keyboard users skip directly to the main content?
